src/base-command.js

```diff
@@ -18,9 +18,13 @@ const yaml = require('js-yaml');
 const path = require('path');
 const libRuntime = require("@adobe/aio-lib-runtime");
 const debug = require('debug')('aio-asset-compute.base');
+const {loadConfig} = require('@adobe/aio-cli-plugin-app/src/lib/config-loader');
```

Comment:
@adobe/aio-cli-lib-app-config now does this.
There is one small change, it exports the loadConfig function directly.
This line becomes 
`const loadConfig = require('@adobe/aio-cli-lib-app-config');`